inc/Functions/MyErrorHandler.php

```diff
@@ -99,7 +99,7 @@ function MyErrorHandler($errno, $errstr, $errfile, $errline)
                 type = 3,
                 description = %string%,
                 sort_tag = "PHP-Fehler"',
-            (int) $auth['userid'],
+            (int) ($auth['userid'] ?? 0),
```

Comment:
Not sure this was supposed to be in here, but fine anyways

Reply:
This is, because otherwise it throws an error that `userid` is not existing. It assumes an authenticated user already.

Reply:
That is a generic underlying issue that DB field definition is either not correct or not correctly applied.

Both can be the case, as there are many locations where the DB definition just assumes that the DB fills in a default value, but sadly also the db update functionality does function correctly and sets NOT NULL on all columns (see https://github.com/lansuite/lansuite/issues/547#issuecomment-1465239882),